cookbook/deployment/heroku.rst

```diff
@@ -217,6 +217,84 @@ application will respond:
 
 You should be seeing your Symfony application in your browser.
 
+Custom Compile Steps
+~~~~~~~~~~~~~~~~~~~~
+
+If you wish to execute additional custom commands during a build, you can leverage
+Heroku's `custom compile steps`_. Let's say for example you want to remove the ``dev``
+front controller from your production environment on Heroku in order to avoid a
+potential vulnerability. Adding a command to remove ``web/app_dev.php`` to Composer's
+`post-install-commands`_ would work, but it also removes the controller in your local
+development environment on each ``composer install`` or ``composer update`` respectively.
+Instead, you can add a `custom Composer command`_ named ``compile`` (this key name is
+a Heroku convention) to the ``scripts`` section of your ``composer.json``. The listed
+commands hook into Heroku's deploy process:
+
+.. code-block:: json
+
+    {
+        "scripts": {
+            "compile": [
+                "rm web/app_dev.php"
+            ]
+        }
+    }
+
+This is also very useful to dump your Assetic assets:
+
+.. code-block:: json
+
+    {
+        "scripts": {
+            "compile": [
+                "app/console assetic:dump"
+            ]
+        }
+    }
+
+Your custom compile steps may depend on node packages, e.g. ``uglifyjs`` or
```

Comment:
as this is about assetic more than custom compile steps, what about putting this in a new section (or a sidebar)?